app/src/main/java/ar/com/wolox/android/training/ui/home/HomeNewsFragment.kt

```diff
@@ -0,0 +1,19 @@
+package ar.com.wolox.android.training.ui.home
+
+
+import ar.com.wolox.android.R
+import ar.com.wolox.wolmo.core.fragment.WolmoFragment
+import ar.com.wolox.wolmo.core.presenter.BasePresenter
+import javax.inject.Inject
+
+class HomeNewsFragment @Inject constructor() : WolmoFragment<BasePresenter<Any>>() {
+
+    val title = "NEWS"
```

Comment:
Should get this from the resource xml in the init method or create a companion object for the constant

Reply:
Solved in next PR update.